jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -87,10 +88,28 @@ then
 		echo "Unable to automatically rebase to branch '${ghprbTargetBranch}'. Please rebase your PR!"
 		exit 1
 	fi
+
+	proceed=0
+
+	read -r -d '' -a FILES_CHANGED < <(git diff --name-only origin/samba-build)
+
+	for i in "${FILES_CHANGED[@]}"
+	do
+		if [[ $i =~ "spec" ]] && [[ $i =~ ${SAMBA_MAJOR_VERS} ]]; then
```

Comment:
mixing `if ...; then` as variation of coding style in this file (`if` on one line, `then` on the next, `for` on one line, `do` on the next).

Reply:
Done.